nltk/corpus/reader/bcp47.py

```diff
@@ -0,0 +1,216 @@
+# Natural Language Toolkit: BCP-47 language tags
+#
+# Copyright (C) 2022 NLTK Project
+# Author: Eric Kafe <kafe.eric@gmail.com>
+# URL: <https://www.nltk.org/>
+# For license information, see LICENSE.TXT
+
+import re
+from warnings import warn
+from xml.etree import ElementTree as et
+
+from nltk.corpus.reader import CorpusReader
+
+
+class Bcp47CorpusReader(CorpusReader):
```

Comment:
```suggestion
class BCP47CorpusReader(CorpusReader):
```
It seems that BCP47 is more fitting.

Reply:
I agree, but since all other corpus reader classes include CorpusReader in their name (see the \_\_all\_\_ list in nltk/corpus/reader/\_\_init\_\_.py), I prefer to do the same for the moment.

Reply:
Apologies, I meant `BCP47CorpusReader`, i.e. capitalising Bcp to BCP.

Reply:
Oh, ok, we can do that, but it will require changes in 3 files.

Reply:
If you'd like, I can handle that, and then merge it afterwards. If you think we're ready for merging afterwards 😄

Reply:
Thanks @tomaarsen, I just made the change.